.pre-commit-config.yaml

```diff
@@ -1,9 +1,9 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 24.10.0
+    rev: 24.3.0
     hooks:
       - id: black
-        args: [--safe]
+        args: ["--line-length", "120", "--diff", "."]
```

Comment:
Running black on "." (root of the repo) is not the best idea, maybe it is better if it runs it for the files that are being committed only.